src/precice/SolverInterface.hpp

```diff
@@ -772,6 +772,47 @@ class SolverInterface {
       int     valueIndex,
       double &value) const;
 
+  /**
+   * @brief setBoundingBox Define a region of interest in order to filter a
+   *        received mesh for a certain mesh region
+   *
+   * @param[in] dataID ID of the Data you want to access through the bounding box
+   * @param[in] boundingBox (axis aligned) which has in 3D the format
+   *            [x_min, x_max, y_min, y_max, z_min, z_max]
+   *
+   * @return ID of the bounding box (similar to a mesh ID)
+   */
+  int setBoundingBox(
+      const int     dataID,
+      const double *boundingBox) const;
```

Comment:
```suggestion
  int setBoundingBox(
      const int     dataID,
      const double *boundingBoxLimits) const;
```
Sth for much later, but before releasing this API, we should generalize to a case where we could also set multiple BB. "setBB" sounds then too much like overwriting.

Reply:
Noted. I will not address it for now, but rather when we have found an overall stable concept.

Reply:
I still think that `boundingBoxLimits` is easier to read here as we already have a `BoundingBox` as  a class.